steamcmd_servers/space_engineers/egg-space-engineers.json

```diff
@@ -0,0 +1,131 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-03-11T23:55:52+01:00",
+    "name": "Space Engineers",
+    "author": "tueye@tuworld.de",
+    "description": "Space Engineers is a voxel-based sandbox game set in space and on planets.",
+    "features": null,
+    "images": [
+        "quay.io\/parkervcp\/pterodactyl-images:debian_wine-5"
+    ],
+    "file_denylist": [],
+    "startup": ".\/start.sh",
```

Comment:
This is a no from me for having it as a start script. Users can edit scripts.

Reply:
The following can all be variables in the panel

WINEARCH=win64
WINEPREFIX=/home/container/.wine
WINEDEBUG=-all wine /home/container/spaceengineers/DedicatedServer64/SpaceEngineersDedicated.exe -path

Reply:
True. Didn't think about the fact that the use of an editable start script may have negative consequences.
With the variables it makes it even a bit nicer. Especially in relation to WINEDEBUG.

Just testing it through again and then pushing the changes in.